### ISSUE TITLE
AppMap explain populates the search bar instead of selecting code objects in the side bar

### ISSUE THREAD

**Issue by ahtrotta** [getappmap/appmap-js#1541]

**Comment by kgilpin** [comment]
Would this happen in response to the user clicking links in the AI response?

**Comment by ahtrotta** [comment]
No, currently we select code objects when we open an appmap in the right column by using the [`setState` method ](https://github.com/getappmap/appmap-js/blob/main/packages/components/src/pages/ChatSearch.vue#L195-L197)and passing it an object with the `selectedObjects` field. Instead we could pass an object with the [`searchBar` field](https://github.com/getappmap/appmap-js/blob/main/packages/components/src/pages/VsCodeExtension.vue#L1040-L1047) to populate the search bar.
